src/validation/tests/statement_validation_tests.rs

```diff
@@ -1577,3 +1577,86 @@ fn while_loop_triggers_error_if_condition_is_not_boolean() {
 
     "###);
 }
+
+#[test]
+fn action_calls_without_parentheses() {
+    // Given a POU with defined actions,
+    // when trying to call them without parentheses
+    let diagnostics = parse_and_validate_buffered(
+        "
+        FUNCTION_BLOCK fb1
+            FOO;
+            BAR;
+        END_FUNCTION_BLOCK
+
+        ACTIONS
+            ACTION FOO
+            END_ACTION
+
+            ACTION BAR
+            END_ACTION
+        END_ACTIONS
+        ",
+    );
+
+    // we expect a validation error for each "call"-statement
+    assert_snapshot!(diagnostics);
```

Comment:
I've experimented with this in another PR of mine, but the snapshots of these validations seem small enough to maybe convert it into a [inline snapshot](https://insta.rs/docs/snapshot-types/#inline-snapshots) (making reviewing and understanding unit tests easier). Do you like the idea, would it make sense here? Should we do this in the future?

Reply:
Good idea, I would do it on new files, but if the files already exist then we plan it for later.